jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/websocket/WebSocketDiagnosticsCollector.java

```diff
@@ -204,6 +207,68 @@ private void uriMismatchWarningCheck(IType type, List<Diagnostic> diagnostics, I
         }
     }
 
+    /**
+     * Creates an error diagnostic if there exists more than one method annotated
+     * with @OnMessage for a given message format.
+     * 
+     * @param type
+     * @param diagnostics
+     * @param unit
+     * @throws JavaModel
+     */
+    private void onMessageWSMessageFormats(IType type, List<Diagnostic> diagnostics, ICompilationUnit unit)
+            throws JavaModelException {
+        IMethod[] typeMethods = type.getMethods();
+        boolean onMessageTextUsed = false;
+        boolean onMessageBinaryUsed = false;
+        boolean onMessagePongUsed = false;
+        for (IMethod method : typeMethods) {
+            IAnnotation[] allAnnotations = method.getAnnotations();
+            for (IAnnotation annotation : allAnnotations) {
+                if (annotation.getElementName().equals(WebSocketConstants.ON_MESSAGE)) {
+                    ILocalVariable[] allParams = method.getParameters();
+                    for (ILocalVariable param : allParams) {
+                        if (!isParamPath(param)) {
+                            String signature = param.getTypeSignature();
+                            String formatSignature = signature.replace("/", ".");
+                            String resolvedTypeName = JavaModelUtil.getResolvedTypeName(formatSignature, type);
+                            if (!resolvedTypeName.equals(WebSocketConstants.SESSION_CLASS)) {
```

Comment:
In the tests, `String resolvedTypeName = JavaModelUtil.getResolvedTypeName(formatSignature, type)` return null to me. See open issue https://github.com/eclipse/lsp4jakarta/issues/265 . If this issue is still occurring, it would be good to apply a null checker for `resolvedTypeName`

Reply:
Just seeing this comment now but that's essentially what I did to fix the issue after running the debugger.